jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/complex/list/ElementsGoogleTests.java

```diff
@@ -0,0 +1,68 @@
+package io.github.epam.html.tests.elements.complex.list;
+
+import com.epam.jdi.light.elements.complex.DataList;
+import io.github.com.custom.Result;
+import io.github.com.custom.SearchResult;
+import io.github.epam.TestsInit;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.*;
+import static io.github.com.pages.Header.*;
+import static io.github.epam.html.tests.site.steps.States.*;
+import static io.github.epam.test.data.ListData.*;
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+import static org.testng.Assert.*;
+
+/**
+ * Created by Roman_Iovlev on 3/2/2018.
+ */
+public class ElementsGoogleTests implements TestsInit {
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        epamLogo.click();
+        search("jdi");
+    }
+    @Test
+    public void validateEntitiesTests() {
+        DataList<SearchResult, Result> jobs = searchPage.search;
+
+        jobs.assertThat(not(empty()))
+            .and(hasSize(greaterThan(2)))
+            .and(hasItem(CORRECT))
+            .and(hasItems(CORRECT, CORRECT_2, CORRECT_3))
+            .and(not(hasItem(CORRUPTED)))
+            .and(not(hasItems(CORRUPTED, CORRUPTED_2)));
+    }
+    @Test
+    public void validateEntities2Tests() {
+        DataList<SearchResult, ?> jobs = searchPage.search2;
+
+        assertEquals(jobs.get(1).name.getText(),"JDI SKYPE");
+        assertEquals(jobs.get(2).name.getText(),"JDI OWNER CONTACT");
+        try {
+            jobs.is().empty();
+            Assert.fail("List should not be empty");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.fail' due to existing static import 'org.testng.Assert.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5485505)